js/functions.js

```diff
@@ -0,0 +1,58 @@
+// 1 задание
+const funcLength = (string, maxLength) => {
+  if(string.length <= maxLength){
+    return true;
+  }
+  return false;
```

Comment:
return обычно отделяется пустой строкой от другого кода
```suggestion

  return false;
```